packages/blockchain-wallet-v4-frontend/src/components/BuySell/Coinify/Create/template.js

```diff
@@ -0,0 +1,96 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import AcceptTerms from './AcceptTerms'
+import VerifyEmail from './VerifyEmail'
+import { FormattedMessage } from 'react-intl'
+import {
+  FaqFormMessage,
+  PartnerHeader,
+  PartnerSubHeader
+} from 'components/IdentityVerification'
+import { MediaContextConsumer } from 'providers/MatchMediaProvider'
+import media from 'services/ResponsiveService'
+
+const CoinifyFaqFormMessage = styled(FaqFormMessage)`
+  left: 60%;
+  top: 10%;
+  margin-left: 20px;
+`
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 20px 15%;
+`
+const SubHeader = styled(PartnerSubHeader)`
+  width: 65%;
+  ${media.mobile`
+    width: 100%;
+  `}
+`
+
+const Create = props => {
+  const { create } = props
+
+  const determineStep = () => {
```

Comment:
declaring functions in SFC is bad for perf and breaking if it returns some component(often causes `componentDidMount` call on every render)

Reply:
this one can obviously be a variably instead of function, because it's never launched after render